pages/rdm_cycle/processing.md

```diff
@@ -3,6 +3,43 @@ title: Processing
 keywords:
```

Comment:
Please, make sure to add contributors' names in the metadata (see page template and editorial board guide).